test/gcm.provider.test.js

```diff
@@ -88,7 +88,7 @@ describe('GCM provider', function() {
 
   describe('for multiple device tokens', function() {
     it('sends Notification as a GCM message', function(done) {
-      var notification = aNotification({ aKey: 'a-value' });
+      var notification = aNotification({ aKey: 'a-value', aNull: null, message: undefined });
```

Comment:
I see, I agree it's a good idea to have that behaviour tested. Please create a new test with a name that makes the intent clear, e.g. "it ignores non-truthy values" instead of modifying an unrelated test.